docs/First-Installation.md

```diff
@@ -21,65 +21,57 @@ The simplest way to get most of these components is through an all-in-one kit su
+![rpi-imager](https://user-images.githubusercontent.com/540857/107868009-493a6400-6e4e-11eb-9272-7f45d569dc44.gif)
 
-To extract from the command line, you can type the following into a Terminal window, placing X with version you downloaded:
+If you don't know which model Raspberry Pi you have, there are a few methods to find out:
 
-`gunzip retropie-4.X.X-rpi2_rpi3.img.gz`
+* The [Raspberry Pi Wikipedia page](https://en.wikipedia.org/wiki/Raspberry_Pi#Connectors) has user-friendly graphics to help determine model by looking at the board itself.
+* If you have SSH/commandline access, run `cat /proc/device-tree/model` and it will output your Pi's model
 
 ### Install
 
-To install the RetroPie SD image on your MicroSD card. (You may need a MicroSD card reader to plug it into your computer) 
-
-1. For Windows you can use [Etcher](https://www.balena.io/etcher/), [Win32DiskImager](http://sourceforge.net/projects/win32diskimager/) or [Raspberry Pi Imager](https://www.raspberrypi.org/software/)
-2. For macOS you can use [Etcher](https://www.balena.io/etcher/), [Apple Pi Baker](https://www.tweaking4all.com/hardware/raspberry-pi/applepi-baker-v2/) or [Raspberry Pi Imager](https://www.raspberrypi.org/software/)
-3. For Linux you can use the `dd` command, [Etcher](https://www.balena.io/etcher/) or [Raspberry Pi Imager](https://www.raspberrypi.org/software/).
-
-See the official Raspberry Pi "WRITING AN IMAGE TO THE SD CARD" [instructions](https://www.raspberrypi.org/documentation/installation/installing-images/).
+1. Click the **Choose SD Card** button. 
+2. The MicroSD card you plugged in earlier should appear in this list. Select it to continue.
+3. Once your "OS" and "SD card" have been chosen, select **Write**, and wait until the operation completes.
+4. Remove the MicroSD card from your PC, slide it back into the slot on your Raspberry Pi, and turn it on.
 
-**Note** RetroPie is built on top of Raspberry Pi OS Buster (a Linux based OS for the Raspberry Pi) and as such the partition on the SD card is EXT4 (a linux filesystem). This partition is is not visible on Windows systems, so the card will show up as a smaller size than usual and you won't be able to see everything on the card, but it is all there. You will be able to access the filesystem over the network as described in the transferring roms section below.
 
-If you're updating from a previous version of RetroPie see [**HERE**](Updating-RetroPie).
+**Note**: You can safely ignore Windows' warnings about this drive. You may see Windows complain that "There is a problem with this drive", but everything is fine. RetroPie is built on top of Raspberry Pi OS Buster (a Linux based OS for the Raspberry Pi) and as such the partition on the SD card is EXT4 (a linux filesystem). This partition is not visible on Windows systems, so the card will show up as a smaller size than usual and you won't be able to see everything on the card. 
```

Comment:
this part is windows-specific. I am not sure it's necessary in any case.

Reply:
The warning doesn't show up with an empty microSD, on first installation. It happens after writing the image or if the microSD has an existing RetroPie/Raspbian. I don't think it's needed.

Reply:
First time I installed Retropie I thought I messed something up and plugged the MicroSD back into my computer, then saw that message on Windows 10. As a result, I followed the message through to what I believe was a GUI version of `chkdsk`. Windows reported the drive was "fixed" (although it probably did nothing), but I was still getting the error and so ignored it. 

I figure having this line could help reduce forum posts about it, but if you aren't getting any to begin with I'll just remove it.